### PR TITLE
Custom Persona Data:URI Bug Fix

### PR DESCRIPTION
**Pull request by derinb**
Custom Persona Data:URI Bug Fix

### CODE REVIEW

**Comment by derinb** [comment]
Andreas, this is very urgent. Because of the data:uri bug, restart version of PP saves personas in preferences with data:uris for headerURL and footerURL, although the custom personas do not work. Because new version uses resource://uri, if there are data:uris in preferences, it is replaced with resource:uris

File: extension/content/customPersonaEditor.js [hunk 1]
```diff
@@ -243,14 +254,14 @@ var CustomPersonaEditor = {
     PersonaService.changeToPersona(this.customPersona);
     switch (PersonaService.appInfo.ID) {
       case PersonaService.THUNDERBIRD_ID:
-	Cc['@mozilla.org/appshell/window-mediator;1'].
+    Cc['@mozilla.org/appshell/window-mediator;1'].
```
Comment (wagnerand) [inline extension/content/customPersonaEditor.js]: Odd formatting.

File: extension/content/customPersonaEditor.js [hunk 2]
```diff
@@ -243,14 +254,14 @@ var CustomPersonaEditor = {
...
         getService(Ci.nsIWindowMediator).
         getMostRecentWindow("mail:3pane").
         document.getElementById("tabmail").
         removeCurrentTab();
         break;
       case PersonaService.FIREFOX_ID:
- 	window.close();
+    window.close();
```
Comment (wagnerand) [inline extension/content/customPersonaEditor.js]: same here.

**Comment by wagnerand** [comment]
Please file a bug for this PR with detailed steps to reproduce the issue to make sure this patch fixes it.

File: extension/content/customPersonaEditor.js (hunk 1 above)
Comment (derinb) [inline extension/content/customPersonaEditor.js]: I am not sure why it is happening but Github Desktop shows they have the same indent. But on Github.com the lines are misaligned.

Actually the file is already contains too many formatting issues. So it has to be prettified all together.

File: extension/content/customPersonaEditor.js (hunk 2 above)
Comment (derinb) [inline extension/content/customPersonaEditor.js]: Same here, Github Desktop shows they have the same indent but Github.com complains.

**Comment by derinb** [comment]
Related bug

https://github.com/mozilla/personas-plus/issues/32

**Comment by wagnerand** [comment]
It's probably because you are mixing tabs and spaces. Text editors have to
interpret how many space one tab represents. So, for some editors it shows
correct but that's just coincidental.

If that file has formatting errors, let's fix that in another patch, but
let's not make things worse.

Am Donnerstag, 9. Juni 2016 schrieb derinb :

**Comment by derinb** [comment]
No, indeed I did not touch those below 2 lines. They got flagged themselves automatically. I tried to reset them but could not succeed. So at least made sure they have the same indent in Github Desktop but this time Github.com cries.

I use Sublime Text and I made sure Spaces 4 is selected. Those two lines were already in a bad format. Maybe there were `/r`

The whole file contains many format issues. So as you suggested I did not prettified it not to make things complex when reviewing.

**Pull request merged**